test/ci_workflow_gen.go

```diff
@@ -239,6 +255,12 @@ func mergeBlankLines(buf *bytes.Buffer) string {
 }
 
 func main() {
+	var err error
+	goJunitReportSHA, err = getRepoHeadSHA1(githubOrg, goJunitReportRepo)
```

Comment:
I made `goJunitReportSHA` a global var, but I could make it a plain `var` and pass it into the `generate*` funcs. Let me know!